### ISSUE TITLE
Request for Data Dictionary

### ISSUE THREAD

**Issue by AldoISED** [paulofelipe/cp2015-jax#2]
Thank you for creating this library, it’s an impressive piece of work! I was wondering if you could provide a data dictionary or documentation that explains the variable names used in the library. Specifically, I’d like to understand how importers and exporters are denoted in the rows and columns of the dataset, as well as how the data provided in Caliendo and Parro (2015) is processed to match the data structure of this library.

**Comment by paulofelipe** [comment]
Hi Aldo,

I have updated the README to detail the data structure. It’s not as clear as I would like yet, but as soon as I have more time, I will create something that makes it easier for anyone who wants to structure new data for the model.

I also removed the variable d_nij_hat from the data file, as the code is not yet suitable for shocks in iceberg costs.

If the data is correct:

- `data["gamma_nkj"].sum(1) + data["gamma_nj"]` should equal 1 for all entries in the resulting matrix.

- `data["pi_nij"].sum(1)` should also equal 1 for all entries in the resulting matrix.
